pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -209,7 +209,7 @@ func (pl *PodTopologySpread) Score(ctx context.Context, cycleState *framework.Cy
 			score += scoreForCount(cnt, c.MaxSkew, s.TopologyNormalizingWeight[i])
 		}
 	}
-	return int64(score), nil
+	return int64(math.Round(score)), nil
```

Comment:
`math.Round` is good to me.  
It is also used in other score plugins like `VolumeBuilding` and `RequestedToCapacityRatioScoring`.